src/components/ChatEntry.js

```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp'
+
+const ChatEntry = (singleEntry) => {
+  const senderName = 
+    singleEntry.sender === 'Vladimir' ? 'remote' : 'local';
+
+  const heart = singleEntry.liked ? '❤️' : '🤍'; 
```

Comment:
The ternaries are great for situations like this right? They are the perfect tool to use when wanting conditionally apply styling!